GetFed/GetFed/Controllers/FoodSearchViewController.swift

```diff
@@ -90,6 +91,22 @@ extension FoodSearchViewController: UITableViewDataSource, UITableViewDelegate {
         selectedIndex = indexPath.row
         performSegue(withIdentifier: SegueID.foodSearchToFoodDetailSegue.rawValue, sender: cell)
     }
+    
+    func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        return true
+    }
+    
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            let foodEntryToDelete = foodArray[indexPath.row]
+            CoreDataManager.sharedManager.deleteEntryByLabel(foodLabel: foodEntryToDelete.label)
+            
+            foodArray.remove(at: indexPath.row)
+            tableView.deleteRows(at: [indexPath], with: .fade)
```

Comment:
You may want to check, but I'm not sure whether this method reload the table view to accurately reflect the state of your data source. If it doesn't, your UI could be out of sync with Core Data.

Reply:
The docs seem to indicate that manually reloading the tableview is not required when using the swipe-to-delete functionality, as long as the row is removed from the data source array and the table view, in that order: (https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/TableView_iPhone/ManageInsertDeleteRow/ManageInsertDeleteRow.html): 

> When the user swipes across a row to display the Delete button for that row, there is a variation in the calling sequence diagrammed in Figure 7-1. When the user swipes a row to delete it, the table view first checks to see if its data source has implemented the tableView:commitEditingStyle:forRowAtIndexPath: method; if that is so, it sends setEditing:animated: to itself and enters editing mode. In this “swipe to delete” mode, the table view does not display the editing and reordering controls. Because this is a user-driven event, it also brackets the messages to the delegate inside of two other messages: tableView:willBeginEditingRowAtIndexPath: and tableView:didEndEditingRowAtIndexPath:. By implementing these methods, the delegate can update the appearance of the table view appropriately.


Thoughts?

Reply:
@jshier Based on the changes, this conversation could be resolved. Thoughts?